From 9d4290832c8fffcd70370065afd445ef8aa3f5fe Mon Sep 17 00:00:00 2001
From: Wentao Zhang <wentao.zhang@windriver.com>
Date: Mon, 22 Aug 2022 13:38:02 +0800
Subject: [PATCH 1/2] Revert upstream commit 9a2a552

The commit 9a2a552 set the Durlify's value
from false to true and added the breaks by setting
the require of the version of less.
The stable version of less on debian is 551-2 now.
Revert this commit as it break the Image builds.

Signed-off-by: Wentao Zhang<Wentao.Zhang@windriver.com>
---
 debian/control | 1 -
 debian/rules   | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/debian/control b/debian/control
index a1fbaae..ddd464b 100644
--- a/debian/control
+++ b/debian/control
@@ -94,7 +94,6 @@ Conflicts: consolekit,
            systemd-shim,
 Breaks: resolvconf (<< 1.83~),
         udev (<< 247~),
-        less (<< 563),
         sicherboot (<< 0.1.6),
 Provides: systemd-sysusers (= ${binary:Version}),
           systemd-tmpfiles (= ${binary:Version}),
diff --git a/debian/rules b/debian/rules
index 06b1e70..f65291a 100755
--- a/debian/rules
+++ b/debian/rules
@@ -70,7 +70,7 @@ CONFFLAGS = \
 	-Dpwquality=false \
 	-Dp11kit=false \
 	-Doomd=true \
-	-Durlify=true \
+	-Durlify=false \
 	-Dgnutls=false \
 	-Dlink-udev-shared=false \
 	-Dlink-systemctl-shared=false \
-- 
2.25.1

